frontend/packages/dev-console/integration-tests/features/topology/topology-chart_area_visual.feature

```diff
@@ -112,8 +112,19 @@ Scenario: Fit to Screen in topology : T-07-TC03
 @regression, @manual
 Scenario: Reset view in topology: T-07-TC02
    Given user has created a workload named "nodejs-ex-git"
-   And user is at the Topolgy page
+   And user is at the Topology page
    When user clicks on Zoom In option
    And user sees the chart area is zoomed
    And user clicks on Reset View option
    Then user sees the chart area is reset to original
+
+@regression
+Scenario: Topology filter by resource: T-07-TC06, T-07-TC07
+   Given user created two workloads with resource type "Deployment" and "Deployment-Config" 
```

Comment:
Can we add this in the background?

Reply:
No, because other test cases have different given condition